_data/remote.yml

```diff
@@ -85,6 +85,15 @@ websites:
       - totp
     doc: http://www.cc-onlinehelp.com/en/#index/administrati/users/authenticati/authenticati.html
 
+  - name: MyFRITZ!
+    url: https://sso.myfritz.net/
+    img: myfritz.png
+    tfa:
+      - totp
+      - hardware
+      - u2f
+    doc: https://en.avm.de/service/fritzbox/fritzbox-7590/knowledge-base/publication/show/3427
```

Comment:
I think that [this doc page](https://en.avm.de/service/fritzbox/fritzbox-7590/knowledge-base/publication/show/3602) might be slightly more useful - it mentions both U2F and TOTP at the bottom of the page and has better details on how to enable